src/components/ClawIcon.tsx

```
@@ -0,0 +1,43 @@
+interface ClawIconProps {
+  size?: number;
+  className?: string;
+  animated?: boolean;
+}
+
+export default function ClawIcon({ size = 48, className = "", animated = true }: ClawIconProps) {
+  return (
+    <div className={`${animated ? "claw-icon" : ""} ${className}`} style={{ width: size, height: size }}>
+      <svg viewBox="0 0 120 120" fill="none" xmlns="http://www.w3.org/2000/svg" width={size} height={size}>
```

Biome (2.3.14)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)